Formula/portable-ruby.rb

```diff
@@ -117,14 +96,12 @@ def install
 
     args << "--with-opt-dir=#{paths.join(":")}"
 
+    system "autoconf"
+
     system "./configure", *args
     make
     make "install"
 
-    # rake is a binstub for the RubyGem in 2.3 and has a hardcoded PATH.
-    # We don't need the binstub so remove it.
-    rm bin/"rake"
```

Comment:
Is this fixed upstream? If so, can we backport the patch instead?

Reply:
Looks like the binary isn't shipped anymore (?) - at least it wasn't there when I tried to delete it. I'll see if I can reproduce, and see if there's a particular commit I can grab.

Reply:
@mistydemeo Probably worth talking through the other comments first to make sure it's worth your time.